CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.8
+
+* Require the latest version of `package:http_parser`.
```

Comment:
I don't like requiring patch versions, even if they fix relevant bugs. There's no reason the version solver should ever select 3.1.0 anymore anyway.

Also, if we did do this, we'd release it as 0.6.7+3, not 0.6.8.